JavaScript/README.md

```diff
@@ -1,9 +1,11 @@
 
-[<-Back to root](../README.md)
+[<-- BACK TO ROOT](../README.md)
 
-# JavaScript
+# **JavaScript**
 
```

Comment:
If in your Java directory you have

```
# Java
## Projects
## Sub-directories
```

You shouldn't skip ## Projects in JS, even if there aren't any projects yet, you could add something like

```
## Projects
- // To do
```

```suggestion
## Projects
- // To do
```